### DISCUSSION TITLE
Export beatmaps on Android

### DISCUSSION THREAD

**Discussion by E8Ib** [ppy/osu#27869] [Ideas]
Until now, osu!lazer on Android devices still can't export beatmaps. (It can export skins! Isn't that amazing?)
Just export to _/storage/emulated/0/Android/data/sh.ppy.osulazer/files/exports_, it's ok.

**Comment by Walavouchey** [comment]
enabling that was tried once in https://github.com/ppy/osu/pull/7977 but it caused a crash, and fixing it has not been a priority

**Reply by Joehuu** [reply]
The code has since changed and doesn't crash anymore. Made a PR in https://github.com/ppy/osu/pull/27987.

**Reply by Joehuu** [reply]
Unmarking answer/solution as accessing exports on the mobile device itself is a requirement before unblocking. The skin exporting on mobile is an oversight, but will stay as it's a waste of time blocking that.
